warehouse/packaging/models.py

```diff
@@ -407,6 +408,14 @@ def urls(self):
 
         return _urls
 
+    @property
+    def github_repo_info_url(self):
+        for parsed in [urlparse(url) for url in self.urls.values()]:
+            segments = parsed.path.strip('/').rstrip('/').split('/')
+            if parsed.netloc == 'github.com' and len(segments) >= 2:
```

Comment:
Would it make sense to change this to something like:

    if parsed.netloc.endswith('github.com') and ...

That way the feature would twig on URLs using `www.github.com/user/repo`, not only `github.com/user/repo`.

Reply:
@bskinn Great catch. Could you file an issue for this? Welcome to take a shot at fixing it as well, let us know if you need any help.

Reply:
Sure thing, filed as #3643. I'll take a shot at putting together a PR for it, too.